core/src/main/java/hudson/security/csrf/CrumbIssuerDescriptor.java

```diff
@@ -64,7 +64,7 @@ public String getCrumbRequestField() {
      */
     public void setCrumbRequestField(String requestField) {
         if (Util.fixEmptyAndTrim(requestField) == null) {
-            crumbRequestField = ".crumb";
+            crumbRequestField = CrumbIssuer.DEFAULT_CRUMB_NAME;
```

Comment:
Even cleaner:

* inline the constant into `setCrumbRequestField`
* move the `System.getProperty` check into `setCrumbRequestField`
* create a new `protected` constructor with no crumb name, to be used from `DefaultCrumbIssuer` and `TestCrumbIssuer`
* make tests just use something like `r.jenkins.getCrumbIssuer().getCrumbRequestField()`